Core/MIPS/x86/X64IRCompALU.cpp

```diff
@@ -185,10 +179,43 @@ void X64JitBackend::CompIR_Compare(IRInst inst) {
 
 	switch (inst.op) {
 	case IROp::Slt:
+		regs_.Map(inst);
+		CMP(32, regs_.R(inst.src1), regs_.R(inst.src2));
+		SETcc(CC_L, R(SCRATCH1));
+		MOVZX(32, 8, regs_.RX(inst.dest), R(SCRATCH1));
+		break;
```

Comment:
This rang a bell so I googled it: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=96846

The recommended sequence here is to xor dest with itself to zero it, then compare and do setcc to dest, rather than setcc + movzx. Although, that's deep in the microoptimization weeds and only works if dst isn't equal to one of the srcs :)

Reply:
And also has the 8-bit register issue once again...

Reply:
Don't bother changing :)